Makefile

```diff
@@ -2,15 +2,15 @@
 #   rsync
 #   wget or curl
 #
-JRUBY_VERSION=1.7.11
-ELASTICSEARCH_VERSION=1.1.1
+JRUBY_VERSION=1.7.17
```

Comment:
Symptoms is:

Bundler 1.9.9 is installed but not available in the current `GEM_PATH`, maybe an issue with gembag.rb